snuba/environment.py

```diff
@@ -32,10 +32,17 @@ def setup_sentry() -> None:
     )
 
 
-clickhouse_rw = ClickhousePool(settings.CLICKHOUSE_HOST, settings.CLICKHOUSE_PORT)
+clickhouse_rw = ClickhousePool(
+    settings.CLICKHOUSE_HOST,
+    settings.CLICKHOUSE_PORT,
+    settings.CLICKHOUSE_USER,
+    settings.CLICKHOUSE_PASS,
+)
 clickhouse_ro = ClickhousePool(
     settings.CLICKHOUSE_HOST,
     settings.CLICKHOUSE_PORT,
+    settings.CLICKHOUSE_USER,
+    settings.CLICKHOUSE_PASS,
     client_settings={"readonly": True},
```

Comment:
You need to merge master in your PR or to rebase, this is now different after https://github.com/getsentry/snuba/pull/860